livre/Eq/FrameData.cpp

```diff
@@ -112,20 +120,23 @@ bool deregisterObject( CoObjectMapPtr objectMapPtr,
 }
 
 FrameData::FrameData( )
-    :   objectFactoryPtr_( new ObjectFactory() ),
-        frameSettingsPtr_( static_cast< FrameSettings* >
-                          ( objectFactoryPtr_->createObject( SMI_FRAME_SETTINGS ) ) ),
-        renderSettingsPtr_( static_cast< RenderSettings* >
-                          ( objectFactoryPtr_->createObject( SMI_RENDER_SETTINGS ) ) ),
-        cameraSettingsPtr_( static_cast< CameraSettings* >
-                          ( objectFactoryPtr_->createObject( SMI_CAMERA_SETTINGS ) ) ),
-        volumeSettingsPtr_( static_cast< VolumeSettings* >
-                          ( objectFactoryPtr_->createObject( SMI_VOLUME_SETTINGS ) ) ),
-        vrParametersPtr_( static_cast< VolumeRendererParameters* >
-                        ( objectFactoryPtr_->createObject( SMI_VR_PARAMETERS ) ) ),
-        prefetchParametersPtr_( static_cast< EFPrefetchAlgorithmParameters* >
-                              ( objectFactoryPtr_->createObject( SMI_EFP_PARAMETERS ) ) )
-
+    : objectFactoryPtr_( new ObjectFactory() )
+    , frameSettingsPtr_( static_cast< FrameSettings* >
+                       ( objectFactoryPtr_->createObject( SMI_FRAME_SETTINGS ) ) )
+    , renderSettingsPtr_( static_cast< RenderSettings* >
+                        ( objectFactoryPtr_->createObject( SMI_RENDER_SETTINGS ) ) )
+    , cameraSettingsPtr_( static_cast< CameraSettings* >
+                        ( objectFactoryPtr_->createObject( SMI_CAMERA_SETTINGS ) ) )
+    , volumeSettingsPtr_( static_cast< VolumeSettings* >
+                        ( objectFactoryPtr_->createObject( SMI_VOLUME_SETTINGS ) ) )
+    , vrParametersPtr_( static_cast< VolumeRendererParameters* >
+                      ( objectFactoryPtr_->createObject( SMI_VR_PARAMETERS ) ) )
+    , prefetchParametersPtr_( static_cast< EFPrefetchAlgorithmParameters* >
+                            ( objectFactoryPtr_->createObject( SMI_EFP_PARAMETERS ) ) )
```

Comment:
It's not whitespaces but commas. Commas must be at the beginning of the lines otherwise it's impossible to use #ifdef in init lists.